### DISCUSSION TITLE
Can I ignore files outside my workspace

### DISCUSSION THREAD

**Discussion by WBare** [streetsidesoftware/vscode-spell-checker#645] [New Feature Requests]
Is it possible to ignore files that get loaded into the editor, but are not in or under my workspace (or my opened directory)?

VS Code Golang will load external files by following calls into libraries elsewhere on the disk, and they often have lots of short names that show up as errors.  These appear in my "problems" tab, but they are obviously not my problems.  Sometimes it makes the problems tab hard to manage when I need the library opened.

It would be nice if I could just never spell check anything outside of my workspace.  I've looked through settings but was not sure how to do it.

Thanks!

**Comment by Jason3S** [comment]
Thank you for asking this.

This will become the default behavior in the next release. There will be an option to enable spell checking files not in the workspace.

**Reply by WBare** [reply]
That is great news.  Thanks very much , Jason.
